jwt_proxy/api.py

```diff
@@ -57,6 +57,18 @@ def validate_jwt(relative_path):
     return response_content
 
 
+@blueprint.route('/fhir/.well-known/smart-configuration')
+def smart_configuration():
+    """Non-secret application settings"""
+
+    results = {
+        "authorization_endpoint": current_app.config.get("OIDC_AUTHORIZE_URL"),
+        "token_endpoint": current_app.config.get("OIDC_TOKEN_URI"),
+        "introspection_endpoint": current_app.config.get("OIDC_TOKEN_INTROSPECTION_URI")
```

Comment:
`black` will get this eventually :)
```suggestion
        "introspection_endpoint": current_app.config.get("OIDC_TOKEN_INTROSPECTION_URI"),
```

Reply:
Ran `black` on all source files - good idea.  We should add to CI when the dust settles :)